.github/workflows/integration-tests.yml

```diff
@@ -74,3 +74,23 @@ jobs:
       - run: pip install -ve .
       - run: pytest ./tests
         working-directory: pyvistaqt
+
+  geovista:
+    name: GeoVista
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: git clone https://github.com/bjlittle/geovista.git --single-branch
+      - name: Install PyVista
+        run: pip install -ve . # pyvista
+      - run: sudo apt-get install libgeos-dev
```

Comment:
@tkoyama010 Note that `geovista` is using `cartopy` as a minor convenience at the moment, and I'm keen to drop that dependency asap. When that happens there won't be any need to install `libgeos-dev` :+1:

Reply:
@bjlittle The download warning of `cartopy` is now an error due to set alerts to capture warnings. We will wait for `cartopy` installation and testing to become an option in `geovista`.

Reply:
Didn't realize that you tripped on warnings by default. I'd suggest downloading cartopy assets prior to running the tests, this should avoid such warnings. I'll push a change to your PR later today, if that's okay?

Reply:
@bjlittle Yes, please. Thank you for your cotribution.

Reply:
> Didn't realize that you tripped on warnings by default

I think its generally a good practice to prevent deprecation warnings from becoming deprecation errors.

We can also just specifically ignore the cartopy warnings. For this integration testing, we mainly just want to catch numpy and pyvista warnings

Reply:
@bjlittle Thank you for the PR of the solution.

@banesullivan We will adopt @bjlittle 's solution this time. If there are more warnings in the future, we will adopt your solution.